docs/setup/forward_proxy.md

```diff
@@ -0,0 +1,71 @@
+# Using a forward proxy with Synapse
+
+You can use Synapse with a forward or outbound proxy. This will be neccssary
+for expamle in corporate environments behind a DMZ (demilitarized zone).
+Synapse supports routing outbound HTTP(S) requests via a proxy. Only HTTP(S)
+proxy is supported, not SOCKS proxy or anything else.
+
+## Configure
+
+The `http_proxy`, `https_proxy`, `no_proxy` environment variable is used to
+specify proxy settings. The environment variable is not case sensitive.
+- `http_proxy`: Proxy server to use for HTTP requests.
+- `https_proxy`: Proxy server to use for HTTPS requests.
+- `no_proxy`: Comma-separated list of hosts, IP addresses, or IP ranges in CIDR
+  format which do not use a proxy and Synapse use a direct connection.
```

Comment:
I find the `no_proxy` description a bit confusing, maybe the following is clearer:

```suggestion
- `no_proxy`: Comma-separated list of hosts, IP addresses, or IP ranges in CIDR
  format which should not use the proxy. Synapse will directly connect to these hosts.
```